src/page-multi-facility/FacilityChart.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+
+import { getFacilities } from "../database";
+import Colors, { MarkColors as markColors } from "../design-system/Colors";
+import Loading from "../design-system/Loading";
+import CurveChartContainer from "../impact-dashboard/CurveChartContainer";
+import { EpidemicModelProvider } from "../impact-dashboard/EpidemicModelContext";
+import useModel from "../impact-dashboard/useModel";
+import {
+  getNewestRt,
+  getRtDataForFacility,
+  isRtData,
+} from "../infection-model/rt";
+import { useLocaleDataState } from "../locale-data-context";
+import { initialPublicCurveToggles } from "./curveToggles";
+import FacilityRowRtValuePill from "./FacilityRowRtValuePill";
+import {
+  useChartDataFromProjectionData,
+  useProjectionData,
+} from "./projectionCurveHooks";
+import { Facility, RtValue } from "./types";
+
+const LoadingWrapper = styled.div`
+  color: ${Colors.opacityGray};
+  display: flex;
+  height: 45%;
+  align-content: center;
+`;
+
+interface FacilityChartProps {
+  facility: Facility | undefined;
+  facilityRtData: RtValue | undefined;
+}
+
+const FacilityChart: React.FC<{
+  scenarioId: string;
+  indexOfFacility: number;
```

Comment:
i'm not so sure there's much benefit to having this passed as a prop? if anything we might want to change the index we use _inside_ the component once the data is loaded (e.g. choose the first facility that has valid data, or the one that is most recently updated - not that I'm necessarily suggesting you do that here, just that it's probably a more likely future use case than arbitrarily choosing a different index from outside this component)